src/routes/Landing.js

```diff
@@ -3,15 +3,18 @@ import React, { useEffect, useState, Fragment } from 'react';
 import { useDispatch } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 import { Button, Modal, Stack, StackItem, Title } from '@patternfly/react-core';
-import Header from '../layout/Header';
-import Body from '../layout/Body';
 import Marketing from '../layout/Marketing';
 import FooterTraditional from '../layout/FooterTraditional';
 import Loading from '../layout/Loading';
 import { activeTechnologies } from '../consts';
 import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/cjs/actions';
 import './Landing.scss';
 
+// Mockup console landing page
+import Body from '../layout/landingPage/Body';
+import Header from '../layout/landingPage/Header';
+import Footer from '../layout/landingPage/Footer';
```

Comment:
I'd _really_ love to not use header, body, footer here because these are not actually those kinds of sections.

rename these to
`FirstPanel`
`SecondPanel`
`ThirdPanel`

that way if we need to add more in the future we can